apps/system-e2e/src/tests/islandis/service-portal/acceptance/social-insurance.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test, BrowserContext, expect } from '@playwright/test'
+import { icelandicAndNoPopupUrl, urls } from '../../../../support/urls'
+import { session } from '../../../../support/session'
+import { label } from '../../../../support/i18n'
+import { messages } from '@island.is/service-portal/health/messages'
```

Comment:
Please un-import and use strings, for now. Playwright doesn't like path-aliases, so this will cause setup issues when testing (at least locally).